tests/e2e/tests/_legacy-experience/woocommerce-blocks/sca-card.spec.js

```diff
@@ -37,7 +37,7 @@ test( 'customer can checkout with a SCA card @smoke @blocks', async ( {
 		.getByRole( 'button', { name: 'Complete' } )
 		.click();
 
-	await page.waitForURL( '**/checkout/order-received/**' );
+	await page.waitForNavigation();
```

Comment:
waitForNavigation() is deprecated, and it may cause race conditions... using waitForURL is the recommended approach.

Reply:
It's not part of this PR; reverted it.